Графика для всех страниц

AleksanrReva opened the pull request:
Подгрузил все изображения в репозиторий, контентные изображения добавил в разметку

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss-individual/1/user/1411729/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/1411729-device-1/3/

AleksanrReva committed:
Исправляет недочёты

Использован правильный атрибут для текста-подсказки в инпуте поиска по сайту.
Исправлен тег для строки адреса в футере.
Переработано содержимое ссылки в подвале страницы на Академию с учётом того, что там будет изображение

AleksanrReva committed:
Добавляет изображения в репозиторий

AleksanrReva committed:
Добавляет изображения в разметку

Подключены контентные изображения. Без задания стилей и проработки alt

AleksanrReva committed:
Добавляет нужные описания изображениям

Kartaviy-arteom commented on images/icons/faficon.svg:
favicon - кстати она в этом формате тут не нужна https://up.htmlacademy.ru/htmlcss-individual/1/criteries#HTML-08

Kartaviy-arteom commented on images/icons/index-icon-action-camera.svg:
тебе не обязательно использовать такие длинные перечисления в наименованиях, по сути это ничего не меняет

Kartaviy-arteom commented on index.html:
```diff
@@ -82,12 +82,14 @@ <h3 class="visually-hidden">Фотографии</h3>
 
           <button class="product-photo-previos" type="button">
             <span class="visually-hidden">Назад</span >
+            <img class="" src="images/icons/icon-left-tall-arrow.svg" width="20" height="60" alt="Назад">
```
декоративная иконка, будет в стилях в разметке быть не должно

Kartaviy-arteom commented on index.html:
```diff
@@ -82,12 +82,14 @@ <h3 class="visually-hidden">Фотографии</h3>
 
           <button class="product-photo-next" type="button">
             <span class="visually-hidden">Вперёд</span >
+            <img class="" src="images/icons/icon-right-tall-arrow.svg" width="20" height="60" alt="Вперёд">
```
и эту убрать

Kartaviy-arteom commented on index.html:
```diff
@@ -21,25 +21,25 @@
-
+            <img class="" src="images/icons/icon-user.svg" alt="" width="13" height="12">
             <a class="navigation-user-link" href="#">Войти</a>
 
           </li>
           <li class="navigation-user-item">
-
+            <img class="" src="images/icons/icon-sort.svg" alt="" width="10" height="12">
             <a class="navigation-user-link" href="#">Сравнить</a>
 
           </li>
           <li class="navigation-user-item">
-
+            <img class="" src="images/icons/icon-cart.svg" alt="" width="16" height="14">
```
эти иконки будут в стилях, из разметки убираем

Kartaviy-arteom commented on index.html:
```diff
@@ -154,32 +156,32 @@ <h2 class="visually-hidden">Категории товаров</h2>
 
           </li>
           <li class="categories-item">
-
+            <img class="" src="images/icons/index-icon-selfie-monopod.svg" width="" height="" alt="">
             <a class="categories-link" href="">Моноподы для селфи</a>
 
           </li>
           <li class="categories-item">
-
+            <img class="" src="images/icons/index-icon-action-camera.svg" width="" height="" alt="">
             <a class="categories-link" href="">Экшн-камеры</a>
 
           </li>
           <li class="categories-item">
-
+            <img class="" src="images/icons/index-icon-fitness-bracelet.svg" width="" height="" alt="">
             <a class="categories-link" href="">Фитнес-браслеты</a>
 
           </li>
           <li class="categories-item">
-
+            <img class="" src="images/icons/index-icon-smart-watches.svg" width="" height="" alt="">
             <a class="categories-link" href="">Умные часы</a>
 
           </li>
           <li class="categories-item">
-
+            <img class="" src="images/icons/index-icon-quadrocopter.svg" width="" height="" alt="">
             <a class="categories-link" href="">Квадрокоптеры</a>
```
эти тоже убираем в стили

Kartaviy-arteom commented on index.html:
```diff
@@ -351,18 +353,21 @@ <h2 class="visually-hidden">Навигация пользователя</h2>
 
           <li class="footer-social-item">
             <a class="footer-social-link" href="#">
+              <img class="" src="images/icons/logo-telegram.svg" width="13" height="11" alt="Телеграм">
               <span class="visually-hidden">Телеграм</span>
             </a>
           </li>
 
           <li class="footer-social-item">
             <a class="footer-social-link" href="#">
+              <img class="" src="images/icons/logo-youtube.svg" width="16" height="10.5" alt="Ютуб">
               <span class="visually-hidden">Ютуб</span>
             </a>
           </li>
 
           <li class="footer-social-item">
             <a class="footer-social-link" href="#">
+              <img class="" src="images/icons/logo-twitter.svg" width="14" height="15" alt="Икс">
```
эти в стили

Kartaviy-arteom commented on index.html:
```diff
@@ -378,7 +383,8 @@ <h2 class="visually-hidden">Контакты</h2>
         <a class="footer-academy-link" href="https://htmlacademy.ru/">
-          .html academy
+          <span class="visually-hidden">.html academy</span>
+          <img src="images/icons/logo-html-academy.svg" width="145" height="36" alt="html academy">
```
тут описание под критерий https://up.htmlacademy.ru/htmlcss-individual/1/criteries#ALLY-01

Kartaviy-arteom commented on catalog.html:
```diff
@@ -21,25 +21,25 @@
-
+            <img class="" src="images/icons/icon-user.svg" alt="" width="13" height="12">
             <a class="navigation-user-link" href="#">Войти</a>
 
           </li>
           <li class="navigation-user-item">
-
+            <img class="" src="images/icons/icon-sort.svg" alt="" width="10" height="12">
             <a class="navigation-user-link" href="#">Сравнить</a>
 
           </li>
           <li class="navigation-user-item">
-
+            <img class="" src="images/icons/icon-cart.svg" alt="" width="16" height="14">
             <a class="navigation-user-link" href="#">Корзина</a>
```
эти иконки уйдут в стили https://up.htmlacademy.ru/htmlcss-individual/1/criteries#IMG-01

Kartaviy-arteom commented on catalog.html:
```diff
@@ -285,16 +285,16 @@ <h3 class="product-card-title">Селфи-палка «Следуй за мно
-      <a class="logo" href="#">
-        <img class="logo-footer" src="#" alt="Логотип проекта «Девайс»">
+      <a class="logo" href="index.html">
+        <img class="logo-footer" src="images/icons/logo-footer.svg" width="145" height="36" alt="device">
```
описание под критерий (точка в конце)

AleksanrReva committed:
Вносит необходимые правки в разметке графики

Фавиконка подключена по Требованию. Имена файлов иконок упрощены. Убраны из разметки, для переноса в стили, иконки. Проработан атрибут alt у соответствующих изображений

Kartaviy-arteom commented on catalog.html:
```diff
@@ -2,14 +2,18 @@
     <meta charset="utf-8">
-    <title>HTML Academy: Каталог</title>
+
+    <link rel="icon" href="images/favicon.ico">
+    <link rel="icon" href="images/favicon.png" type=" type="image/png">
+
```
.ico должна лежать в корне проекта, второй ионке не хватает размеров.
Подробнее в критерии https://up.htmlacademy.ru/htmlcss-individual/1/criteries#HTML-08

Kartaviy-arteom commented on images/favicon.ico:
переместить в корень проекта
